build/VSIX.targets

```diff
@@ -1,13 +1,14 @@
 <Project>
```

Comment:
@natemcmaster - is it worth trying to move this set of targets to aspnet/BuildTools given that we're going to add another VSIX in the future?

Reply:
Given that we just spun our last build for 15.5 this is a safe time to churn stuff. We won't need to build from `dev` for a little while.

Reply:
Yes. It'll require a little more design, but I'm thinking it is worth providing a way for a repo to say "I need to build X and Y with MSBuild.exe, not dotnet-msbuild".